src/service/impl/graphql-client.ts

```diff
@@ -0,0 +1,26 @@
+import ApolloClient, { DefaultOptions } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
+import fetch from 'cross-fetch';
+
+const link = new HttpLink({
+  uri: 'https://api.entur.io/journey-planner/v2/graphql',
+  fetch
```

Comment:
Maybe we should reuse the http agent that we use for the sdk? We should at least provide an agent with keepAlive enabled and sane socket timeouts.

Reply:
There are some TypeScript collisions with node-fetch as that uses it's own definitions with a different "agent" implementation. I started switching it out, but require some testing on keepalive so decided to await some.